package.json

```diff
@@ -75,6 +75,7 @@
         "wsrun": "^5.2.4"
     },
     "resolutions": {
-        "merkle-patricia-tree": "3.0.0"
+        "merkle-patricia-tree": "3.0.0",
+        "**/bignumber.js": "^9.0.2"
```

Comment:
only way I could get it to reconcile all the hoisted 9.0.1s :shrug:

Reply:
I think it's probably fine 🙈  I guess the biggest risk is that some third-party dep either depends on a much older version (or in the future a newer version) and breaks

Reply:
There actually already are some deps that hoist an older version but it doesn't seem to break anything.